.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
```

Comment:
There is no Lua VM available for travis, and the python one is not needed. The ruby template is [their default](https://docs.travis-ci.com/user/ci-environment/#ruby-aka-common-vm-images), and probably most minimal vm/container.